curricula/models.py

```diff
@@ -560,7 +560,7 @@ def clone(self, attrs={}, commit=True, m2m_clone_reverse=True, exclude=[]):
 """
 
 
-class Chapter(Page, RichText, CloneableMixin):
+class Chapter(InternationalizablePage, RichText, CloneableMixin):
```

Comment:
Is this the base class to be using going forward for other new content types? In other words, is there any reason not to use InternationalizablePage if we don't actually plan to translate the content any time soon?

Reply:
Good to know, thanks